zipline/pipeline/factors/factor.py

```diff
@@ -1530,3 +1615,20 @@ def demean(row):
 
 def zscore(row):
     return (row - nanmean(row)) / nanstd(row)
+
+
+def winsorize(row, limits, inclusive):
+    if isinstance(limits, int) or isinstance(limits, float):
```

Comment:
Rather than doing this logic on every row we process, we should just convert our args into the format expected by `scipy_winsorize` at term construction time in `Factor.winsorize`.